### DISCUSSION TITLE
Different Stems

### DISCUSSION THREAD

**Discussion by rpgrunwald** [intel/openvino-plugins-ai-audacity#118] [Ideas]
I've been using demucs to separate tracks and I most frequently want just an isolated bass track and then a complete track without bass.  The command line for that is:  demucs "08 Show Me.mp3" --mp3 -n htdemucs_ft --jobs 8 -v --two-stems bass  gives me two MP3 files. 

Any way to do that with openvino?

Also, using demucs with pytorch_cuda is an order of magnitude faster than openvino, is there anything I can do to enable cuda?

**Comment by RyanMetcalfeInt8** [comment]
Hi @rpgrunwald,

You can produce the two tracks you're interested in by:
1. Performing music separation in 4-stem mode.
2. Select the drums, vocals, and 'other' track.
3. From the 'Tracks' menu, choose 'Mix and Render'.

Now you'll have 'bass-only' and a 'bassless' track.

> Also, using demucs with pytorch_cuda is an order of magnitude faster than openvino, is there anything I can do to enable cuda?

CUDA isn't well supported by OpenVINO right now, so not really.
